pkg/util/sys/device.go

```diff
@@ -116,6 +116,10 @@ type LocalDisk struct {
 	Encrypted bool `json:"encrypted,omitempty"`
 }
 
+func (d *LocalDisk) String() string {
+	return fmt.Sprintf("LocalDisk{Name: %s,	Parent: %s,	HasChildren: %t, DevLinks: %s, Size: %d, UUID: %s, Serial: %s, Type: %s, Rotational: %t, Readonly: %t, Partitions: %v, Filesystem: %s, Vendor: %s, Model: %s, WWN: %s, WWNVendorExtension: %s, Empty: %t, CephVolumeData: %s, RealPath: %s, KernelName: %s: Encrypted: %t ", d.Name, d.Parent, d.HasChildren, d.DevLinks, d.Size, d.UUID, d.Serial, d.Type, d.Rotational, d.Readonly, d.Partitions, d.Filesystem, d.Vendor, d.Model, d.WWN, d.WWNVendorExtension, d.Empty, d.CephVolumeData, d.RealPath, d.KernelName, d.Encrypted)
```

Comment:
```suggestion
	return fmt.Sprintf("%+v", *d)
```

See https://play.golang.org/p/hH5nLfTS5gg